dd-sdk-android/src/test/kotlin/com/datadog/android/utils/forge/ForgeExt.kt

```diff
@@ -44,6 +44,9 @@ internal fun Forge.exhaustiveAttributes(
         .toMap().toMutableMap()
     map[""] = anHexadecimalString()
     map[aWhitespaceString()] = anHexadecimalString()
+    map[anAlphabeticalString()] = aMap {
+        anAlphaNumericalString() to anAlphaNumericalString()
```

Comment:
Here you're making a  `Map<String, String>`, it would be interesting to have a `Map<String,Any>` with: 
```kotlin
    map[anAlphabeticalString()] = aMap {
        anAlphaNumericalString() to anElementFrom(
            anAlphaNumericalString(),
            aBoolean(),
            anInt(),
            …
        )
    }
```

Reply:
at least I would add a `Map` with more than 1 level to test the recursion.

Reply:
Makes sense, indeed. Updated PR.